tests/macho/test_builder.py

```diff
@@ -13,6 +13,10 @@
 
 lief.logging.set_level(lief.logging.LEVEL.INFO)
 
+def align_to(value, alignment):
+    # llvm::alignTo
+    return (value + alignment - 1) & ~(alignment - 1)
```

Comment:
Feel free to backport this implementation in the `utils.hpp` as well which is absolutely not optimized

Reply:
I did consider this, but in the end I decided that I don't want to do it.
The current implementation in `utils.hpp` allows alignment to any number, however this code from LLVM supports alignment only for powers of two.
That's why `utils.hpp` should be changed separately and carefully, ensuring that none uses this function with an argument that is not power of two.

And I just realized I need to add an assert that `alignment` is power of two here.

Reply:
Make sense!